src/nav/apm.yml

```diff
@@ -549,6 +549,8 @@ pages:
                 path: /docs/apm/agents/nodejs-agent/installation-configuration/es-modules
               - title: New Relic CodeStream integration for Node.js
                 path: /docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration
+              - title: Enable source maps support
```

Comment:
i can't access this path and i think it's because the enable-source-maps is lacking a .mdx extension?

Reply:
It looks like the .mdx extension was the solution.